DESCRIPTION

```diff
@@ -80,5 +80,5 @@ Remotes:
     mrc-ide/dust,
     mrc-ide/eigen1,
     mrc-ide/mcstate,
-    mrc-ide/odin,
```

Comment:
this branch pin can be removed now, we've updated that. Increase the required version of odin above to 1.5.0 though (and also in the script that does the compilation)